check_graphite

```diff
@@ -31,8 +32,10 @@ EXIT_WARNING    = 1
 EXIT_CRITICAL   = 2
 EXIT_UNKNOWN    = 3
 
+requests.packages.urllib3.disable_warnings()
```

Comment:
i just wanted to surpress the InsecureRequestWarning when i am using a selfsigned SSL Cert